node/pkg/vaa/structs.go

```diff
@@ -58,6 +59,15 @@ type (
 	}
 
 	SignatureData [65]byte
+
+	TransferPayloadHdr struct {
+		Type         uint8
+		TokenChainID ChainID
+		TokenAddress Address
+		ToChainID    ChainID
+		ToAddress    Address
+		Amount       *big.Int
```

Comment:
Shouldn't this be a https://github.com/holiman/uint256 instead? (we've used this type elsewhere)

Reply:
it would be nicer if these fields appeared in the same order as they do in the binary payload